wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,58 @@
+# Restricted Mode
+
+A restriction is a timeout from the community forced upon accounts that exhibit abnormal, suspicious, or rule-breaking behaviour.
+
+When an account is restricted, it is unable to interact with the community until the restriction is lifted. Restricted players can still continue playing and submitting scores, but their profile will be invisible to others.
+
+The following features are disabled for restricted accounts:
+- in-game chat
+- private messaging on the forums
+- posting on forums
+- updating maps or profile details
+- store purchases
+
+## My account has been restricted! What do I do?
+
+If your account has been restricted, do not panic. Please follow these steps:
+- **Wait 24 hours.** Some restrictions are the results of automated processes. As we check every new restriction daily, please wait 24 hours - if your restriction was erroneous, it will be lifted within that time.
+- **Contact our support team.** If your account remains restricted after that time, please send an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) to ask about the reasons for it. We will need at the very least to know your username to be able to check! Remember to send the email from the email address attached to your account so that we can confirm you are the owner.
+
+In the cases where an account becomes restricted for breaking the [rules](/wiki/rules), a cooldown of three months (strictly) will be given before the user is allowed to send an appeal. If the account you asked about is not your first, please check if your original account is restricted or not. If it is not, you can continue playing on it, but make sure you do not continue accessing your multi-accounts as that might lead to further punishments.
```

Comment:
`If your account has become restricted for breaking the [rules](/wiki/rules), a strict cooldown of three months will be enforced before you are allowed to send an appeal. [newline]`

Reply:
`If your original account is not restricted, you may log into it and continue playing normally. but make sure not to log into your multi-accounts from then on, as doing so may lead to further punishment.`